src/device/GPU_Device.hh

```diff
@@ -41,7 +42,7 @@ namespace rtt_device {
  *         << "\n" << endl;
  *     for( size_t device=0; device<numDev; ++device )
  *         gpu.printDeviceSummary( device, out );
- *   
+ *
  *   // Print the message to stdout
  *   cout << out.str();
  * \endcode
```

Comment:
It appears there is an autodoc error for this file: `Preprocessing /home/home/travis/Draco/src/device/GPU_Device.hh:52: error: included file device/test/gpu_hello_rt_api.cc is not found. Check your EXAMPLE_PATH (warning treated as error, aborting now)`.  Should this example file be suffixed with `.cu`?

Reply:
I fixed this.  I also added a real call to a cuda kernel in the example/test.

Reply:
The test looks great!